read_replica/features/data.spec.ts

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs'
+import sa from 'superagent'
+
+import { suite, test } from '@testdeck/jest'
+import { Severity } from 'allure-js-commons'
+
+import { description, feature, log, severity, step } from '../.jest/jest-custom-reporter'
+import { Hooks } from './hooks'
+
+@suite('api-gateway')
+class Data extends Hooks {
+  @feature('Read Replica')
+  @severity(Severity.CRITICAL)
+  @description('Check if REST GET requests are succeeding')
+  @test
```

Comment:
```suggestion
  @params({ db: 'rr', error: null, public: true }) \\ error can be 'Read replica does not handle POST requests for the path /rest/v1/countries at the moment.' in another test for example 
  @params({ db: 'all', error: null, public: true })
  @params({ db: 'primary', error: null, public: true })
```

wdyt about parametrizing these tests to send these requests to all options: rr, load-balancer, primary?
should be as easy as ^ and one `if` in the `expect(error).toBeNull() and data to be ...` or `expect(error).toBeDefined() and error.message to be ...`

Reply:
if this is too much, can be added later of course